app/server/src/env.d.ts

```diff
@@ -1,14 +1,15 @@
 declare global {
     declare namespace NodeJS {
         interface ProcessEnv {
-            PORT: string;
+            PORT: 'production' | 'development' | 'staging';
```

Comment:
Did you mean NODE_ENV for this? I don't think the PORT should ever be any of those values.

Reply:
Ah, yes I did, great catch!

Reply:
fixed.